pkg/stargate/deployments.go

```diff
@@ -17,8 +19,10 @@ import (
 )
 
 const (
-	cassandraConfigDir  = "/config"
-	cassandraConfigPath = "/config/cassandra.yaml"
+	cassandraConfigDir     = "/config"
+	cassandraConfigPath    = "/config/cassandra.yaml"
+	CassandraConfigMap     = "generated-cassandra-config"
+	UserCassandraConfigMap = "cassandra-config"
```

Comment:
`UserCassandraConfigMap` seems unused.

Reply:
right. I must have removed its use in one of the iterations on the code.